internal/store/postgres/asset_repository.go

```diff
@@ -598,12 +610,15 @@ func (r *AssetRepository) insert(ctx context.Context, ast *asset.Asset) (string,
 	return id, nil
 }
 
-func (r *AssetRepository) update(ctx context.Context, assetID string, newAsset, oldAsset *asset.Asset, clog diff.Changelog) error {
+func (r *AssetRepository) update(ctx context.Context, tx *sqlx.Tx, newAsset, oldAsset *asset.Asset, clog diff.Changelog) error {
+	assetID := oldAsset.ID
```

Comment:
is there any case where the assetID changed?

Reply:
should be not. Its UUID anyway, so ideally we use URN instead for identifier from outside.

And actually assetID came from fetchedAsset https://github.com/goto/compass/blob/7acbcda18f9d36d755ca7f5eccc879501fb03f2d/internal/store/postgres/asset_repository.go#L341 which it got from existing asset (old asset) by GetByURN, so I just refactor it to remove redundancy